text.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <main>
+        <header class="header">
+            <div class="logo">Jo</div>
+            <div></div>
+            <div class="menu">
+                <a href="#" class="nav">
+                    <div ></div>
+                    <div ></div>
+                    <div ></div>
+                </a>
+            </div>
+        </header>
+        <section class="headline">
```

Comment:
- [ ] I can see that the contents of your headline section are overlapping each other due to the fact that you added a fixed height (i,e height: 30%;) to the headline texts. I would recommend you remove the fixed height and use margin instead to add the spacing 👍🏽.  Kindly ensure your design match the Figma design you chose. You can check out the Figma design specification for the exact properties used👍🏽 

| Figma Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/63473584/191637749-8fa6ba7e-a58d-42ac-97dc-5eb41fa87b32.png)  | ![image](https://user-images.githubusercontent.com/63473584/191637816-65ede2a1-0c5f-4682-9188-40905ed85e83.png)  |